internal/datacoord/server.go

```diff
@@ -534,19 +543,28 @@ func (s *Server) initMeta(chunkManager storage.ChunkManager) error {
 	if s.meta != nil {
 		return nil
 	}
-	etcdKV := etcdkv.NewEtcdKV(s.etcdCli, Params.EtcdCfg.MetaRootPath.GetValue())
+	s.watchClient = etcdkv.NewEtcdKV(s.etcdCli, Params.EtcdCfg.MetaRootPath.GetValue())
+	metaType := Params.MetaStoreCfg.MetaStoreType.GetValue()
+	log.Info(fmt.Sprintf("data coordinator connecting to metadata store %s.", metaType))
```

Comment:
`log.Info("data coordinator connecting to metadata store",  zap.String("metaType", metaType))`

Reply:
Nice